applications/fluid_structure_interaction_precice/perpendicular_flap/fluid-exadg/input.json

```diff
@@ -0,0 +1,23 @@
+{
+    "General": {
+        "Precision": "double",
+        "Dim": "2",
+        "IsTest": "false"
+    },
+    "SpatialResolution": {
+        "DegreeFluid": "2",
+        "DegreeStructure": "3",
+        "RefineFluid": "1",
+        "RefineStructure": "3"
```

Comment:
Here and in the other file "solid-exadg" it seems that we have redundant information. Is this precice-related (see also the other comment on why either the fluid or the solid driver is instantiated in the case of the precice FSI solver) or could we remove this redundancy? Generally speaking, I think we should precisely adapt the input files to the solver at hand.

Reply:
The problem in the current implementation: the `Fluid` parameter here as well as the `Structure` parameter are required by configuration ( `has_to_be_set = true`).

Reply:
I will make a suggestion in another PR how we could resolve this

Reply:
what about this issue?

Reply:
Resolved :)